openstack/loadbalancer/v2/pools/requests.go

```diff
@@ -122,6 +127,27 @@ type CreateOpts struct {
 	// or false (DOWN).
 	AdminStateUp *bool `json:"admin_state_up,omitempty"`
 
+	// A list of ALPN protocols. Available protocols: http/1.0, http/1.1, h2.
+	AlpnProtocols []string `json:"alpn_protocols,omitempty"`
```

Comment:
I think it would follow common practice in this repo to create `type AlpnProtocol string` and use it similarly to [`RuleDirection`](https://github.com/gophercloud/gophercloud/blob/master/openstack/networking/v2/extensions/security/rules/requests.go#L91)? This would apply for all create/update objects.

Reply:
```go
type AlpnProtocol string

const (
    ProtocolHTTP10 AlpnProtocol = "http/1.0"
    ProtocolHTTP11 AlpnProtocol = "http/1.1"
    ProtocolH2 AlpnProtocol = "h2"
)

type CreateOpts struct {
    // ...
    AlpnProtocols []AlpnProtocol `json:"alpn_protocols,omitempty"`
    // ...
}
```